signac/core/h5store.py

```diff
@@ -261,16 +261,29 @@ class H5Store(MutableMapping):
             assert 'foo' in h5s
             assert h5s.foo == 'bar'
             assert h5s['foo'] == 'bar'
+
+    :param filename:
+        The filename of the underlying HDF5-file. This file will be create if it
+        does not exist yet with the standard open mode ('a').
```

Comment:
Is this part about the standard open mode redundant? Or am I missing something?

Reply:
I wanted to make sure that users understand they don't *have* to create the file beforehand. The comment that 'a' is default may be redundant, but is that a problem?

Reply:
To me, this implies that if you specify `mode='r'` and the file does not yet exist, it overrides your provided `'r'` and uses `'a'` instead. I'm not sure if that is true.

Reply:
Got it, will revise.